ethereum/spec/src/main/java/tech/pegasys/teku/spec/util/BeaconStateUtil.java

```diff
@@ -557,14 +565,13 @@ void processDepositWithoutCheckingMerkleProof(
 
       // Verify the deposit signature (proof of possession) which is not checked by the deposit
       // contract
-      if (BLS_VERIFY_DEPOSIT) {
+      if (verifyDeposits) {
```

Comment:
Nice to get rid of this.  We'll need to make sure it's handled right in the fuzzing classes when they get switched over.